### ISSUE TITLE
Popper: modifier "applyStyles" provided an invalid "fn" property, expected "function" but got "undefined"

### ISSUE THREAD

**Issue by zenodallavalle** [react-bootstrap/react-bootstrap#6019]
## Describe the bug

Copy-pasted the docs popover example, run with `yarn start` but console is telling me this:

> Popper: modifier "applyStyles" provided an invalid "fn" property, expected "function" but got "undefined" 

Strange thing is that this is not happening with `yarn build`. Plus is not happening using codesandbox.

## To Reproduce

- `create-react-app test`
- `yarn add react-bootstrap@next bootstrap@5.1.0`
- `yarn start`
- in App.js place:
```
import 'bootstrap/dist/css/bootstrap.min.css';
import { OverlayTrigger, Popover, Button } from 'react-bootstrap';

function App() {
  return (
    <div>
      <OverlayTrigger
        trigger='click'
        placement='auto'
        overlay={
          <Popover id='popover-positioned-auto'>
            <Popover.Header as='h3'>Popover example</Popover.Header>
            <Popover.Body>
              <strong>Holy guacamole!</strong> Check this info.
            </Popover.Body>
          </Popover>
        }
      >
        <Button>Popover example</Button>
      </OverlayTrigger>
    </div>
  );
}

export default App;
```

## Steps to reproduce the behavior:

1. Run with `yarn start`.
2. Click on the Popover example button to show popover.
3. Click again to hide it and check console.

## Reproducible Example

This is not happening in codesandbox environment.

## Environment

- Operating System: Window 10 x64 build 19042
- Node v14.15.4
- Browser: Chrome, 92.0.4515.159
- React-Bootstrap Version: 2.0.0-beta.6
- Bootstrap Version 5.1.0

**Comment by jerome-diver** [comment]
i don't know if this can have a relation, but two days ago i had problem with @popperjs/core. After that, an issue was posted and last @popperjs/core version (2.10.1) fixed the issue. Maybe look at this... because it requier @opperjs/core instead of popperjs...

**Comment by zenodallavalle** [comment]
@jerome-diver  thank you for answering. Just tested with @popperjs/core v2.10.1 and @restart/ui version 0.2.2, error still remain.

**Comment by fastlorenzo** [comment]
@kyletsang just released v0.2.3 of @restart/ui, you can upgrade to that version to have this fixed (thanks!)

`yarn add @restart/ui@^0.2.3` with yarn
